### PR TITLE
Use underlying device dynamically in MiniCPMV2_6 and Mllama models

### PR DESCRIPTION
**Pull request by vshekhawat-hlab**
## Motivation

The current implementation explicitly sets the device to cuda, which limits flexibility and compatibility across different hardware environments. To make the models device-agnostic, the change dynamically selects the underlying device using get_device().

## Modifications

MiniCPMV2_6:
Imported get_device from sglang.srt.utils.
Replaced the hardcoded cuda device with get_device() when moving the resampler to the target device.

MllamaForConditionalGeneration:
Imported get_device from sglang.srt.utils.
Replaced device="cuda" with device=get_device() when creating tensors for batched_ar_ids.

## Checklist

- [x] Format your code according to the [Code Formatting with Pre-Commit](https://docs.sglang.ai/references/contribution_guide.html#code-formatting-with-pre-commit).
- [ ] Add unit tests as outlined in the [Running Unit Tests](https://docs.sglang.ai/references/contribution_guide.html#running-unit-tests-adding-to-ci).
- [ ] Update documentation / docstrings / example tutorials as needed, according to [Writing Documentation](https://docs.sglang.ai/references/contribution_guide.html#writing-documentation-running-docs-ci).
- [ ] Provide throughput / latency benchmark results and accuracy evaluation results as needed, according to [Benchmark and Profiling](https://docs.sglang.ai/references/benchmark_and_profiling.html) and [Accuracy Results](https://docs.sglang.ai/references/accuracy_evaluation.html).
- [ ] For reviewers: If you haven't made any contributions to this PR and are only assisting with merging the main branch, please remove yourself as a co-author when merging the PR.
- [ ] Please feel free to join our Slack channel at https://slack.sglang.ai to discuss your PR.

### CODE REVIEW

File: python/sglang/srt/models/mllama.py [hunk 1]
```diff
@@ -835,7 +835,7 @@ def _batch_image_inputs(self, forward_batch: ForwardBatch):
                 dtype=torch.float32,
             )
             batched_ar_ids = torch.ones(
-                bs, max_num_images, dtype=torch.int64, device="cuda"
+                bs, max_num_images, dtype=torch.int64, device=get_device()
```
Comment (merrymercy) [inline python/sglang/srt/models/mllama.py:838]: cache this

Comment (merrymercy) [inline python/sglang/srt/models/mllama.py:888]: cache this

**Review by merrymercy — CHANGES_REQUESTED** [review]

File: python/sglang/srt/models/mllama.py (hunk 1 above)
Comment (vshekhawat-hlab) [reply python/sglang/srt/models/mllama.py:888]: > cache this

get_device method is already with lru_cache.
